gettsim/benefits/arbeitsl_geld.py

```diff
@@ -22,16 +22,20 @@ def ui(
         soli_st_params,
         beit_bem_grenz=soz_vers_beitr_params[f"rvmaxek{westost}"],
         werbungs_pausch=eink_st_abzuege_params["werbungskostenpauschale"],
-        soz_vers_pausch=params["soz_vers_pausch"],
+        sozial_versich_pauschale_alg=params["sozial_versich_pauschale_alg"],
```

Comment:
We should use `arbeitsl_geld` as identifier.

Reply:
corrected the argument's name to  `sozial_versich_pauschale`, because it's used also used for Elterngeld.

Reply:
In any case, we should stick to one abbreviation. I just saw the yaml file and orientated, therefore, my guess on the identifier.